src/api/app/controllers/webui/announcements_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Webui::AnnouncementsController < Webui::WebuiController
+  def show
```

Comment:
I think this should be merged into `AnnouncementsController.show` in a `respond_to` block for HTML...

Reply:
@hennevogel I've tried this but announcement routes are declared inside `APIMatcher` and it seems it accepts only xml and json format. If I'm not wrong, I should declare something like `def self.public_or_about_path?(request)` for exceptions.

So I wonder if it is really a good idea... :thinking:

Reply:
There are two 'sections' of routes in our app, they are based on requested content format.

One part is based on `APIMatcher` (basically all requests that want `:xml`) and the other one on `WebuiMatcher` (basically requests for every other format than `:xml`).

You would need to setup the show route (to the same controller) in both sections.

Reply:
Ok, done, now the routes respond to html too. But I have to specify which layout it has to use because this controller is not under WebuiController, the one that sets the layout (choosing between webui and webui2).

Is this still worth it? @hennevogel  :thinking:

Reply:
Hm that indeed is a problem we have to solve. Okay, let's figure this out afterward.